draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -717,6 +737,175 @@ Steps:
 23. Output (x, y)
 ~~~
 
+### Boneh-Franklin Method {#supersingular}
+
+The map2curve_bf(alpha) implements the Boneh-Franklin method {{BF01}} which
```

Comment:
Stupid question: on what page is this method described in the paper?

Reply:
Secction 5.2 of "Identity-Based Encryption from the Weil Pairing" Boneh, Franklin